diff -urN gnome-keyring-manager-0.0.3.orig/gnome-keyring-manager.desktop.in.in gnome-keyring-manager-0.0.3/gnome-keyring-manager.desktop.in.in
--- gnome-keyring-manager-0.0.3.orig/gnome-keyring-manager.desktop.in.in	2004-11-17 13:34:07.517253736 +0100
+++ gnome-keyring-manager-0.0.3/gnome-keyring-manager.desktop.in.in	2004-11-17 13:34:32.275489912 +0100
@@ -10,4 +10,4 @@
 X-GNOME-Bugzilla-Product=gnome-keyring-manager
 X-GNOME-Bugzilla-Component=general
 X-GNOME-Bugzilla-Version=@VERSION@
-Categories=GNOME;Application;System;Utility;
+Categories=GTK;GNOME;Utility;
